frontend/lib/src/components/elements/ArrowTable/ArrowTable.tsx

```diff
@@ -142,13 +142,27 @@ function generateTableCell(
   const { type, content, contentType } = table.getCell(rowIndex, columnIndex)
   const styledCell = getStyledCell(table, rowIndex, columnIndex)
 
-  const formattedContent =
+  let formattedContent =
     styledCell?.displayContent || formatArrowCell(content, contentType)
+  let hasStylerTooltip: boolean = false
 
   const style: React.CSSProperties = {
     textAlign: isNumericType(contentType) ? "right" : "left",
   }
 
+  if (
+    formattedContent &&
+    formattedContent.endsWith(`<span class="pd-t"></span>`)
+  ) {
+    // This is a bit hacky, but to support the Pandas Styler's tooltip feature,
+    // we need to convert the specific HTML element (used for tooltips) from
+    // the display value into an actual span element.
+    formattedContent = formattedContent.replace(
+      `<span class="pd-t"></span>`,
```

Comment:
Do you think it would be best to use a regex to replace it only if it is at the end of `formattedContent`? I doubt it will ever make a difference, but it makes the code consistent with the check for ``formattedContent.endsWith(`<span class="pd-t"></span>`)``.
```suggestion
      /<span class="pd-t"><\/span>$/,
```

Reply:
Yes, that should work as well 👍 Changed it